cypress/e2e/a11y.cy.js

```diff
@@ -0,0 +1,39 @@
+/// <reference types="Cypress" />
+import '../support/e2e.js'
+
+describe('A11y Test Suite', () => {
+    beforeEach(() => {
+        cy.visit('/')
+        cy.injectAxe()
+    })
+    
+    context('Light Mode', ()=>{
+    it('finds no a11y issues in light mode', () => {
```

Comment:
```suggestion
      it('finds no a11y issues in light mode', () => {
```

Atencão com a indentação.